geom/src/distance.rs

```diff
@@ -222,3 +241,25 @@ impl Default for Distance {
         Distance::ZERO
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn round_up_for_axis() {
+        for (input, expected) in [
```

Comment:
I don't know if this is the intuitive rule for rounding up or not. Maybe once we hit 0.1 miles, we should go up to the nearest 0.1 of a mile, and then nearest mile, 10 miles, etc.

Also this is always in terms of feet because of #692